src/loaders/sqlClient.js

```diff
@@ -12,7 +12,7 @@ module.exports = async () => {
 
   const knexConfig = (await knexfile())[config.clusterEnv];
 
-  logger.log(`SQL endpoint at: ${knexConfig.connection.host}`);
+  logger.log(`sql endpoint at: ${knexConfig.connection.host}`);
```

Comment:
why not uppercase? SQL is an abbreviation, so it should be all caps

Reply:
Since we ended up using lowercase for `sqlClient` and other folders, I added this change for coherency